app/src/main/java/com/sdp/movemeet/view/chat/ChatActivity.java

```diff
@@ -318,11 +319,10 @@ public void onResume() {
     public void sendMessage(View view) {
         String userName = fullNameString;
         String messageText = messageInput.getText().toString();
-        Message message = new Message(userName, messageText, userId, null /* no image */);
+        Message message = new Message(userName, messageText, userId, "no imageUrl" /* no image */, Long.toString(new Date().getTime()));
```

Comment:
if `"no imageUrl"` is a value you'll be comparing against in the future, to check whether an image is attached, it should be put into a static constant

Reply:
Yes that's a good point! This will make the code cleaner and more ogranized! Thanks for this suggestion!